thumbor/point.py

```diff
@@ -49,19 +49,19 @@ def __init__(
         weight=1.0,
         origin="alignment",
     ):
-        self.x = int(x)  # pylint: disable=invalid-name
-        self.y = int(y)  # pylint: disable=invalid-name
-        self.height = int(height)
-        self.width = int(width)
+        self.x = x  # pylint: disable=invalid-name
+        self.y = y  # pylint: disable=invalid-name
+        self.height = height
+        self.width = width
         self.weight = weight
         self.origin = origin
 
     @classmethod
     def from_square(
         cls, x, y, width, height, origin="detection"
     ):  # pylint: disable=invalid-name
-        center_x = x + width // 2
-        center_y = y + height // 2
+        center_x = x + (width // 2)
+        center_y = y + (height // 2)
```

Comment:
```suggestion
        center_x = x + width // 2
        center_y = y + height // 2
```

Reply:
hey @scorphus !
thanks for the suggestion. It was already implemented.